### PR TITLE
chore: revise button content

### PR DESCRIPTION
**Pull request by rossmoody**
## Description

The PR revises the content on the <Button> component page to UXCS approved copy. 

## Checklist

- [ ] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [ ] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [ ] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [ ] :zap: audited via [web.dev](https://web.dev/measure/) for performance and SEO
- [ ] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

**Comment by rossmoody** [comment]
@jzempel @allisonacs Added a MediaButton example to the lineup and borrowed the copy from Input. Worth another look.

File: src/examples/button/ButtonMedia.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Button } from '@zendeskgarden/react-buttons';
+import { ReactComponent as LeafIcon } from '@zendeskgarden/svg-icons/src/16/leaf-stroke.svg';
+
+const Example = () => (
+  <Row alignItems="center">
```
Comment (jzempel) [inline src/examples/button/ButtonMedia.tsx:14]: [nit] `alignItems` is not needed for same-sized items.

Comment (rossmoody) [reply src/examples/button/ButtonMedia.tsx:14]: TY!

**Comment by rossmoody** [comment]
@allisonacs This one needs your review

**Pull request merged**